app/services/asp/mappers/adresse_mapper.rb

```diff
@@ -27,7 +27,7 @@ def codetypeadr
       end
 
       def localiteetranger
-        student.address
+        student.address[..37]
```

Comment:
c'est pourquoi qu'on limite l'adresse à 37 charactères ? Contrainte de l'ASP ?

Reply:
yes; en vrai ils acceptent pas encore les localité étrangères mais pour qu'on se simplifie la vie de notre côté on obéit quand même à leur MAXLENGTH